spotbugsTestCases/src/java/ghIssues/issue574/Issue574XmlElement.java

```diff
@@ -0,0 +1,37 @@
+package ghIssues;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlValue;
+
+/**
+ * Tests that "URF_UNREAD_FIELD" warning is suppressed for fields with @XmlElement, @XmlAttribute, and @XmlValue
```

Comment:
`@` in javadoc is treated as a javadoc tag. It is better to wrap by the code tag like `{@code @FooBar}`, or the link tag like `{@link FooBar}`.